types/mathjs/index.d.ts

```diff
@@ -1,22 +1,22 @@
-// Type definitions for mathjs
+// Type definitions for mathjs 3.20
```

Comment:
mathjs 4.0 is out now, but I do not believe these types are completely correct for 4.0